cdap-docs/developer-manual/source/cloud-runtimes/profiles/assigning-profiles.rst

```diff
@@ -8,5 +8,75 @@
 Assigning Profiles
 ==================
 
-Describe how to assign a profile
+Profiles can be assigned to batch pipelines in a variety of ways. You can assign a default
+profile for all of CDAP as well as a default profile for a specific namespace. You can also
+assign a profile to a batch pipeline that will be used for runs that are manually started,
+and you can assign a profile to a pipeline schedule.
 
+The profile that is actually used at runtime is determined by walking the CDAP hierarchy
+from most general to most specific.
+
+instance > namespace > schedule/manual run
```

Comment:
Isn't the precedence in the opposite order?

Reply:
ok i'll reverse the order and say from most specific to most general.